src/applications/verify/containers/VerifyApp.jsx

```diff
@@ -51,46 +78,44 @@ export class VerifyApp extends React.Component {
     }
   }
 
-  renderVerifyButton(signInMethod) {
-    const verifyWithLoginGov =
-      this.signinMethodLabels.logingov === signInMethod;
-
-    const renderOpts = {
-      copy: verifyWithLoginGov ? 'Login.gov' : 'ID.me',
-      renderImage: verifyWithLoginGov ? (
-        <LoginGovSVG />
-      ) : (
-        <img
-          role="presentation"
-          aria-hidden="true"
-          alt="ID.me"
-          src="/img/signin/idme-icon-white.svg"
+  verifyButton(signInMethod) {
+    if (
+      signInMethod === this.signinMethodLabels.idme ||
+      signInMethod === this.signinMethodLabels.logingov
+    ) {
+      const selectCSP = signInCSP =>
+        this.renderOpts.find(csp => csp.copy === signInCSP);
+      const { className, copy, renderImage, policy } = selectCSP(signInMethod);
+      return (
+        <VerifyButton
+          key={copy}
+          className={className}
+          copy={copy}
+          renderImage={renderImage}
+          policy={policy}
         />
-      ),
-      className: `usa-button ${
-        verifyWithLoginGov ? 'logingov-button' : 'idme-button'
-      }`,
-    };
-
-    return (
-      <button className={renderOpts.className} onClick={() => verify()}>
-        <strong>
-          Verify with <span className="sr-only">{renderOpts.copy}</span>
-        </strong>
-        {renderOpts.renderImage}
-      </button>
-    );
+      );
+    }
+    return this.renderOpts.map(({ copy, renderImage, className, policy }) => (
+      <VerifyButton
+        key={copy}
+        className={className}
+        copy={copy}
+        renderImage={renderImage}
+        policy={policy}
+      />
+    ));
   }
 
   render() {
     const { profile } = this.props;
-    const signInMethod =
-      this.signinMethodLabels[(profile?.signIn?.serviceName)] || 'ID.me';
+    const signInMethod = this.signinMethodLabels[
+      (profile?.signIn?.serviceName)
+    ];
```

Comment:
Let's change this to do the lookup via the `SERVICE_PROVIDERS` instead of the `signinMethodLabels`.